### ISSUE TITLE
Struggle with dictionaries

### ISSUE THREAD

**Issue by Janaue** [Eloston/ungoogled-chromium#1073]
I have w10 1809 up to date, I use chrlauncher and I have ungoogled chromiuim 83.0.4103.97
I use a stock profile. I go to pastebin.com and type ''above lune luna del sol '' to get various words in various language and yet I do not get the red underline indicating the words ae wrong.
I have french on the computer right now and the language offered for spellchecking are french, french again and english and none shows the red line...

The dictionaries I need are US english, French (classical) and spanish.

**Comment by wchen342** [comment]
Dis you try the method [here](https://github.com/Eloston/ungoogled-chromium/issues/1030#issuecomment-626586130)?

**Comment by Eloston** [comment]
I wrote up another method here: https://github.com/Eloston/ungoogled-chromium/issues/188#issuecomment-643665850

**Comment by Janaue** [comment]
With chrlaunche I installer a official chrome and the ungoogled chrome in 2 different folders
From the official chrome, chrome is installed in french and I enable the english dictionary, so I get a folder Dictionary with en-US-9-0.bdic in it.

For the ungoogled version, chrome is in french again and I enable again the english dictionary. I close the browser and put the en-US-9-0.bdic form the official folder to the ungoogled dictionary folder.
I launch the ungoogled chrome and for spell checking,  I get to choose between ''all languages '', ''french'', ''french'' again , ''english''
-when all languages are set to be corrected, there is no correction at all in pastebin.com
eg the string '' lumiere soir soie silk moon esperanza mooon'' has no red underline
-same thing when the first french is selected [this selects also the second french]
-same thing when the second french is selected [this selects also the first french]
-I get the correct underlines when i choose the english spell checking.

In the official version, everything works well.

I wanted to get official chrome to DL the french .bdic, so I removed the french language for the UI, I displayed chrome in spanish, then restarted, then added french again, but I did not get a french .bdic 

To get the french.bdic in the ungoogled version, I installed portable iridium https://iridiumbrowser.de/downloads/windows and it DLed the french and english dictionaries. This time the spellchekcer works correctly. I still have two ''french'' in submenu for the spellchecking in the ungoogled version.
